### DISCUSSION TITLE
Add an option to not stop rain after sleep

### DISCUSSION THREAD

**Discussion by DerpDerpling** [PurpurMC/Purpur#601] [__Completed__] [answered]
i saw this mod today while browsing curseforge and thought it would be a nice feature to be able to toggle. a lot of my players dont like when its raining/storming and someone sleeps, then the rain stops. this feature would keep the rain going even if someone sleeps.
https://www.curseforge.com/minecraft/mc-mods/keep-the-rain

**Comment by katekerllenevich** [comment, marked as answer]
added in 612b1b9831f9aa18596eeb8e6276190bbfa248d9
